plugins/qeta-backend/src/utils/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { subDays, format } from 'date-fns';
+
+export const stringDateTime = (dayString: string) => {
+  const dateTimePeriod = Number(dayString.toString().slice(0, -1));
```

Comment:
This is a bit unclear what this does? Could the period be renamed to "from" and also you could add "until" parameter?

Reply:
The idea was to simplify it a bit and be able to say only how many days you want to get.

Example: period=10d -> data from the last 10 days, so the function transforms 10d => 2023-05-13

But we can use the from/until pattern too 😉